_posts/blog/2019-04-25/2019-04-25-how-to-sell-services-for-cryptocurrency-on-openbazaar.markdown

```diff
@@ -0,0 +1,73 @@
+---
+title: "How to <span class='post-title-extra'>Sell Services for Cryptocurrency</span> on OpenBazaar"
+layout: post
+date: '2019-04-25 08:30:00 -0600'
+social_title: 'How To Sell Services for Cryptocurrency on OpenBazaar'
+social_description: 'On OpenBazaar you can easily sell services for cryptocurrency. Here’s how.'
+---
+
+Making sales is about more than just moving physical products. People also make entire careers—and often very high level careers!—out of selling the services they perform. These can be things done from anywhere in the world like like web design, software development and copywriting, or very local things like medical assistance, legal representation and lawn care.
```

Comment:
Are those hyphen (-) or em-dashes(—)? They should be em-dashes if you're using them to break apart a sentence.

Reply:
They're totally emdashes but they seem so small coming through VS Code??